src/main/java/nl/dtls/fairdatapoint/api/controller/CatalogController.java

```diff
@@ -0,0 +1,144 @@
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package nl.dtls.fairdatapoint.api.controller;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import nl.dtl.fairmetadata4j.io.MetadataException;
+import nl.dtl.fairmetadata4j.model.CatalogMetadata;
+import nl.dtl.fairmetadata4j.model.FDPMetadata;
+import nl.dtl.fairmetadata4j.utils.MetadataUtils;
+import nl.dtls.fairdatapoint.service.metadata.MetadataServiceException;
+import org.eclipse.rdf4j.model.IRI;
+import org.eclipse.rdf4j.rio.RDFFormat;
+import org.springframework.data.rest.webmvc.ResourceNotFoundException;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.ModelAndView;
+import springfox.documentation.annotations.ApiIgnore;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.UUID;
+
+import static nl.dtls.fairdatapoint.util.ThrowingFunction.suppress;
+
+@RestController
+@Api(description = "FDP metadata")
+@RequestMapping("/fdp/catalog")
+public class CatalogController extends MetadataController {
+
+    /**
+     * Get catalog metadata
+     *
+     * @param id
+     * @param request
+     * @param response
+     * @return Metadata about the catalog in one of the acceptable formats (RDF Turtle, JSON-LD, RDF
+     * XML and RDF N3)
+     * @throws IllegalStateException
+     * @throws MetadataServiceException
+     */
+    @ApiOperation(value = "Catalog metadata")
+    @RequestMapping(value = "/{id}", method = RequestMethod.GET, produces = {"text/turtle",
+            "application/ld+json", "application/rdf+xml", "text/n3"})
+    @ResponseStatus(HttpStatus.OK)
+    public CatalogMetadata getCatalogMetaData(@PathVariable final String id,
+                                              HttpServletRequest request, HttpServletResponse response) throws
+            MetadataServiceException, ResourceNotFoundException {
```

Comment:
This indentation style is prone to cause inconsistencies across different IDEs, we can fallback to a agreed-upon style enforced by the IDEs to guard against it. See the style discussion in the review comments.

Reply:
Resolved by https://github.com/FAIRDataTeam/FAIRDataPoint/pull/53#issuecomment-536993575